src/OrchardCore.Modules/OrchardCore.Resources/Services/ResourcesTagHelperProcessor.cs

```diff
@@ -0,0 +1,60 @@
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using OrchardCore.ResourceManagement;
+
+namespace OrchardCore.Resources.Services;
+
+public class ResourcesTagHelperProcessor : IResourcesTagHelperProcessor
```

Comment:
Why you didn't extend `TagHelper` instead?

Reply:
This service is equally used by `ResourcesTagHelper` and the `ResourcesTag` Liquid filter.
Arguably the Liquid filter is used more in production. So extending `TagHelper` would be unnecessary complexity and overhead.